bigquery/tests/system.py

```diff
@@ -659,14 +662,93 @@ def test_sync_query_w_standard_sql_types(self):
                 'expected': [{u'_field_1': [1, 2]}],
             },
         ]
-        for example in EXAMPLES:
+
+    def test_sync_query_w_standard_sql_types(self):
+        examples = self._generate_standard_sql_types_examples()
+        for example in examples:
             query = Config.CLIENT.run_sync_query(example['sql'])
             query.use_legacy_sql = False
             query.run()
             self.assertEqual(len(query.rows), 1)
             self.assertEqual(len(query.rows[0]), 1)
             self.assertEqual(query.rows[0][0], example['expected'])
 
+    def test_dbapi_w_standard_sql_types(self):
+        examples = self._generate_standard_sql_types_examples()
+        for example in examples:
+            Config.CURSOR.execute(example['sql'])
+            self.assertEqual(Config.CURSOR.rowcount, 1)
+            row = Config.CURSOR.fetchone()
+            self.assertEqual(len(row), 1)
+            self.assertEqual(row[0], example['expected'])
+            row = Config.CURSOR.fetchone()
+            self.assertIsNone(row)
+
+    def _load_table_for_dml(self, rows, dataset_name, table_name):
+        import csv
+        from google.cloud._testing import _NamedTemporaryFile
+
+        dataset = Config.CLIENT.dataset(dataset_name)
+        retry_403(dataset.create)()
+        self.to_delete.append(dataset)
+
+        greeting = bigquery.SchemaField(
+            'greeting', 'STRING', mode='NULLABLE')
+        table = dataset.table(table_name, schema=[greeting])
+        table.create()
+        self.to_delete.insert(0, table)
+
+        with _NamedTemporaryFile() as temp:
+            with open(temp.name, 'w') as csv_write:
+                writer = csv.writer(csv_write)
+                writer.writerow(('Greeting'))
+                writer.writerows(rows)
+
+            with open(temp.name, 'rb') as csv_read:
+                job = table.upload_from_file(
+                    csv_read,
+                    source_format='CSV',
+                    skip_leading_rows=1,
+                    create_disposition='CREATE_NEVER',
+                    write_disposition='WRITE_EMPTY',
+                )
+
+        def _job_done(instance):
+            return instance.state.lower() == 'done'
+
+        # Retry until done.
+        retry = RetryInstanceState(_job_done, max_tries=8)
+        retry(job.reload)()
+        self._fetch_single_page(table)
+
+    def test_sync_query_w_dml(self):
+        dataset_name = _make_dataset_name('dml_tests')
+        table_name = 'test_table'
+        self._load_table_for_dml([('Hello World',)], dataset_name, table_name)
+
+        query = Config.CLIENT.run_sync_query(
+            'UPDATE {}.{} '
+            'SET greeting = \'Guten Tag\' '
+            'WHERE greeting = \'Hello World\''.format(
+                dataset_name, table_name))
```

Comment:
This is a lot to process. Can you set up the template separately from the formatting? Also, feel free to use double quotes here to make the text easier to read (your call though).